<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomeScreen</class>
 <widget class="QWidget" name="HomeScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>423</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="InsulinGauge" name="InsulinGaugeWidget" native="true"/>
       </item>
       <item>
        <widget class="QComboBox" name="graphWindowSelector">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>50</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>1 Hour</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3 Hours</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>6 Hours</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QWidget" name="graphWidget" native="true">
       <property name="maximumSize">
        <size>
         <width>800</width>
         <height>400</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="optionsButton">
       <property name="text">
        <string>Options</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bolusButton">
       <property name="text">
        <string>BOLUS</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>InsulinGauge</class>
   <extends>QWidget</extends>
   <header>InsulinGauge.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>bolusButton</sender>
   <signal>clicked()</signal>
   <receiver>HomeScreen</receiver>
   <slot>navigateToBolus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>256</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>optionsButton</sender>
   <signal>clicked()</signal>
   <receiver>HomeScreen</receiver>
   <slot>navigateToOptions()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>256</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>navigateToOptions()</slot>
  <slot>navigateToBolus()</slot>
 </slots>
</ui>
